rviz_rendering/src/rviz_rendering/ogre_logging.cpp

```diff
@@ -96,13 +100,12 @@ namespace rviz_rendering
 {
 
 OgreLogging::Preference OgreLogging::preference_ = OgreLogging::NoLogging;
-// TODO(wjwwood): refactor this to not have static members.
-std::string OgreLogging::filename_;  // NOLINT: cpplint doesn't allow static strings
+const char * OgreLogging::filename_ = "";
 
 void OgreLogging::useLogFile(const std::string & filename)
 {
   preference_ = FileLogging;
-  filename_ = filename;
+  filename_ = filename.c_str();
```

Comment:
This is not safe, as the lifetime of `filename` is not guaranteed to be longer than `filename_`, that's why I just disable the linter on this one until that is resolved some other way. You can either leave the NOLINT or fix it another way.